src/client/components/Searchbar.tsx

```diff
@@ -1,8 +1,9 @@
-import React, { useEffect, useState } from "react";
```

Comment:
I think this file is not properly merged.
Could you compare with the current committed file and fix it?
https://raw.githubusercontent.com/Takashicc/decafe/main/src/client/components/Searchbar.tsx